h/browser/chrome/manifest.json.jinja2

```diff
@@ -45,5 +45,9 @@
     "lib/*",
     "help/*",
     "content/web/viewer.html"
-  ]
+  ],
+
+  "externally_connectable": {
```

Comment:
I reckon the default behaviour should be to generate an extension that is *not* externally connectable, rather than to generate one which allows connections from `https://hpt.is`.